xbmc/ApplicationPlayer.cpp

```diff
@@ -115,6 +115,15 @@ bool CApplicationPlayer::OpenFile(const CFileItem& item, const CPlayerOptions& o
       return true;
     }
   }
+  else if (player && player->m_name != newPlayer)
+  {
+    CloseFile();
+    if (player->m_name != newPlayer)
```

Comment:
Does CloseFile() change player->m_name ?

Reply:
You mean, cause it is checked again?

Reply:
In short: Neither PAPlayer nor VideoPlayer change the Player name, when CloseFile is called on them, but CloseFiles has a "reopen" argument ... 

I think we don't need this to be checked again. @FernetMenta what do you think?